store/resource.js

```diff
@@ -10,6 +10,10 @@ class Resource {
 
   static defaultValues = {}
 
+  get retrieved() {
```

Comment:
I decided to move this check to the separate method because it's shared with a descendent class now but I feel this method misleading: it's not clear the resource itself is loaded or the resource including all subresources is loaded (retrieved). I am keed to leave this as it is because I have no better option but I would appreciate any idea.